docs/man-openmpi/man3/MPI_Alloc_mem.3.rst

```diff
@@ -77,8 +77,8 @@ pointer to a pointer of arbitrary type (e.g., ``void **``).
 Fortran NOTES
 -------------
 
-The :ref:`MPI_Alloc_mem` calls requirse the use of the ``iso_c_binding`` module
-for due to the use of ``TYPE(C_PTR)``.
+The :ref:`MPI_Alloc_mem` calls requires the use of the ``iso_c_binding`` module
```

Comment:
This should actually be `require`, not `requires`.

Sorry, my initial comment said "requires"; I fixed it a minute or two later.